### PR TITLE
feat: autocomplete Fields in form view

### LINKED ISSUE #5913: 🔦 Feature: add autocomplete in Forms input fields
### Please confirm if feature request does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

I was thinking to use nocodb shared Form view for "subscribe to our Newsletter" page. But after a quick test I realized that even though I specified input as "email" there wasn't autocomplete="email" (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) assigned, which would be really valuable for "subscribe to Newsletter" page

### Suggested Solution

either allow for user to set autocomplete attribute, or autodetect it and auto assign it for simple case (like email)

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by DarkPhoenix2704**
## Change Summary

Provide summary of changes with issue number if any.
Closes #5913 

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by wingkwong** [comment]
Rerunning playwright tests

File: packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue [hunk 1]
```diff
@@ -169,10 +167,7 @@ const onDecode = async (scannedCodeValue: string) => {
                         :data-testid="`nc-form-input-cell-${field.label || field.title}`"
                         :class="`nc-form-input-${field.title?.replaceAll(' ', '')}`"
                         :column="field"
-                        :edit-enabled="editEnabled[index]"
-                        @click="editEnabled[index] = true"
-                        @cancel="editEnabled[index] = false"
-                        @update:edit-enabled="editEnabled[index] = $event"
+                        :edit-enabled="true"
```
Comment (wingkwong) [inline packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: What is the reason changing that?

**Review by wingkwong — COMMENTED** [review]
For `Email` and `Url`, can you provide some screenshots to show autocomplete is working in both case? So far I've tested a bit. Didn't see it works.

File: packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue (hunk 1 above)
Comment (DarkPhoenix2704) [reply packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: It makes the Cell component to render the input element by default. Removes unwanted compute.

Comment (wingkwong) [reply packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: if we don't need these in all types of forms, you may also update `Form.vue` and `packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/survey.vue`.

**Comment by DarkPhoenix2704** [comment]
https://github.com/nocodb/nocodb/assets/64391274/408abeea-73cb-4a25-a335-6d0691111d4b

https://github.com/nocodb/nocodb/assets/64391274/72ce8f9e-53ea-409a-97d9-f6b2bfa7f8f0

File: packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue (hunk 1 above)
Comment (wingkwong) [reply packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: survey.vue not yet updated

Comment (wingkwong) [reply packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: https://github.com/nocodb/nocodb/blob/d575c87316529c8a13e86669fe717b1b5cd93154/packages/nc-gui/pages/index/%5BtypeOrId%5D/form/%5BviewId%5D/index/survey.vue#L303-L306

Comment (DarkPhoenix2704) [reply packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:170]: Updated

**Review by wingkwong — COMMENTED** [review]
1. The tests failed constantly - I think they may be caused due to the changes. Please double check.
2. Wanna have @pranavxc reconfirm that the below code is redundant for all types of form.

```
:edit-enabled="editEnabled[index]"
@click="editEnabled[index] = true"
@cancel="editEnabled[index] = false"
@update:edit-enabled="editEnabled[index] = $event"
```

**Pull request closed without merging**